src/App.js

```diff
@@ -1,26 +1,22 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
Please do your best to fit your code to the provided rules instead of disabling them.